src/components/banner.tsx

```diff
@@ -5,11 +5,11 @@ import React, { useState, useEffect } from "react";
 // This is a lazy way of doing things but will work until
 // we put a more robust solution in place.
 //
-const SHOW_BANNER = false;
+const SHOW_BANNER = true;
 const BANNER_TEXT =
-  "Live Workshop @ Vue.JS Live (10/19 @ 9am PT) on Vue web performance tips/tricks.";
+  "Live Workshop with Power Home Remodeling (11/18 @ 10am PT) on streamlining issue management and communication.";
 const BANNER_LINK_URL =
-  "https://ti.to/gitnation/vuejs-conf-remote-only-2021/with/fmw0pknt-84?utm_medium=banner&utm_source=sentry&utm_campaign=event-vuejs-2021";
+  "https://sentry.io/resources/streamline-issue-management-with-powerhrg/?utm_medium=banner&utm_campaign=event-powerHRG-workshop-2021";
```

Comment:
@dorothymlee, this is the URL from your PR 4343. You can just commit this change on this PR rather than merging two PRs. If you have any questions, feel free to ping me. 

```suggestion
  "https://sentry.io/resources/streamline-issue-management-with-powerhrg/?promo_name=docs-banner";
```

Reply:
Hi @imatwawana - apologies! I just committed the the URL to this PR with Rahul. Let me know if that helps.

Reply:
No worries! I just wanted to save you the time of waiting for two PRs to merge. Looks good to me.